### ISSUE TITLE
lcd.screensize() returns (320, 320)

### ISSUE THREAD

**Issue by MinoruInachi** [m5stack/micropython-m5stack#3]
Firmware: m5cloud-20180102-v0.2.5.bin

LCD's catalog spec is 2.0 inch TFT-LCD 320x240, but
MicroPython API lcd.screensize() returns (320, 320).

```
MicroPython ESP32_LoBo_v2.0.8 - 2017-11-04 on M5Stack with ESP32
Type "help()" for more information.
>>> from m5stack import lcd
>>> lcd.screensize()
(320, 320)
```

**Comment by 0x1abin** [comment]
Hi, lcd.screensize bug will fix at next version.
Think you

**Comment by MinoruInachi** [comment]
Thanks.
I confirmed that lcd.screensize () was fixed in v0.3.0 firmware.
However, lcd.winsize () still returns (320, 320) by default.

```python3
>>> from m5stack import lcd
>>> lcd.screensize()
(320, 240)
>>> lcd.winsize()
(320, 320)
>>> lcd.resetwin()
>>> lcd.winsize()
(320, 320)
```

**Comment by MinoruInachi** [comment]
I confirmed that lcd.winsize () was fixed in v0.3.2 firmware.
Thanks.
